style.css

```diff
@@ -0,0 +1,209 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+.headline-bar {
+  height: 72px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-links {
+  display: none;
+}
+
+.nav-links.open {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #6070ff;
+  height: 100%;
+  width: 100%;
+}
+
+.nav-links ul {
+  display: flex;
+}
+
+.nav-links.open ul {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+}
+
+.nav-links ul li {
+  margin-right: 10px;
+}
+
+.nav-links.open ul li {
+  margin-top: 1.5rem;
+}
+
+.nav-links ul li a {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.nav-links.open ul li a {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.toogle {
+  height: 45px;
+  width: 45px;
+  transition: all 0.5s ease-in-out;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+
+.menu-bar {
+  width: 30px;
+  height: 4px;
+  position: relative;
+  background-color: #344563;
+  border-radius: 5px;
+  transition: all 0.5s ease-in-out;
+}
+
+.toogle.open .menu-bar {
+  transform: translateX(-45px);
+  background-color: transparent;
+}
+
+.menu-bar::before,
+.menu-bar::after {
+  content: "";
+  position: absolute;
+  width: 30px;
+  height: 4px;
+  border-radius: 5px;
+  background-color: #344563;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-bar::before {
+  transform: translateY(-10px);
+}
+
+.toogle.open .menu-bar::before {
+  transform: rotate(45deg) translate(35px, -35px);
+  background-color: #fff;
+}
+
+.menu-bar::after {
+  transform: translateY(10px);
+}
+
+.toogle.open .menu-bar::after {
+  transform: rotate(-45deg) translate(35px, 35px);
+  background-color: #fff;
+}
+
+.logo h3 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  margin: 0 1rem;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.intro-paragraph,
+.social-media-btn {
+  margin: 0 1rem;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./images/header-shapes mobile.png");
+  background-repeat: no-repeat;
+}
+
+.intro-paragraph h1 {
```

Comment:
- [ ] Kindly go accordingly to the Figma template.
- [ ] Line height font size needs to be reduced.

<img width="1412" alt="Screen Shot 2022-10-29 at 09 09 42" src="https://user-images.githubusercontent.com/81444338/198841727-a4af0d21-cef0-4620-895e-ce73d408bde8.png">